datahub-web-react/src/app/entity/shared/tabs/Dataset/Schema/utils/useDescriptionRenderer.tsx

```diff
@@ -32,7 +32,6 @@ export default function useDescriptionRenderer(editableSchemaMetadata: EditableS
                         },
                     }).then(refetch)
                 }
-                editable
```

Comment:
wouldn't this make dataset schemas not editable? I think we need to pass this in via context or else we wont be able to share this logic btwn datasets & glossary terms

Reply:
https://github.com/linkedin/datahub/pull/3813/files#diff-a628caf403fe8ddae8c9934ae260353cd62c9c4613bab82f90ce9b7fe0d04ffa
i have already added a context for this

Reply:
will update this component with context

Reply:
Sorry, i am little bit confused
I have already used ` const isSchemaEditable = React.useContext(SchemaEditableContext);`  in `SchemaDescriptionField.tsx`
and used the `<SchemaEditableContext.Provider>` in `SchemaTab.tsx`
wouldnt that be enough to share the editing logic between datasets and glossary terms , or i need to do some more modification ?

Reply:
oh I see- I was confused. Thanks!